### DISCUSSION TITLE
Adding Kotlin Support?

### DISCUSSION THREAD

**Discussion by ghost** [nesbox/TIC-80#1325] [Ideas]
I would like to work in kotlin.

**Comment by nesbox** [comment]
As know Kotlin is a very big thing, you have to embed the whole Java VM to the project, not sure it's possible for the TIC project.
Thanks.

**Reply by msx80** [reply]
Eventually you could just load the installed JVM shared libraries. If there's no installed jvm, an error could be raised.
Just saying, i still think adding jvm support is a bit overkill.
